src/plugin-sidebar.js

```diff
@@ -183,6 +185,26 @@ const CreateBlockThemePlugin = () => {
 										<Icon icon={ chevronRight } />
 									</HStack>
 								</NavigatorButton>
+
+								<hr></hr>
```

Comment:
I think this should be a `CardDivider` too, rather than two `hr`s.

Reply:
It seems like CardDivider is intended to be used inside a Card component. I replaced the <hr/> with Divider components instead: https://github.com/WordPress/create-block-theme/commit/93ff377ee2cf54492b8e1e474a33c3c60ff9ff78